docs/standard/analyzers/api-analyzer.md

```diff
@@ -34,16 +46,31 @@ The **Error List** window contains warnings with a unique ID per deprecated API,
 
 By clicking on the ID, you go to a webpage with detailed information about why the API was deprecated and suggestions regarding alternative APIs that can be used.
 
-Any warnings can be suppressed by right-clicking on the highlighted member and selecting **Quick Actions and Refactorings**. There are two ways to supress warnings: 
+Any warnings can be suppressed by right-clicking on the highlighted member and selecting **Suppress *diagnostic ID***. There are two ways to supress warnings: 
 
 * locally (in source)
 * globally (in a suppression file)
 
+### Suppresing warnings localy
+
+To suppress warnings localy you need to right-click on the member you want to suppress warnings for and select **Suppress *diagnostic ID***, then click **in Source**. Your code will be framed with `#pragma` like it is shown below:
+!["Screenshot of code framed with #pragma warning disable"](media/api-analyzer/suppress-in-source.jpg)
+
+### Suppresing warnings globally
+
+To suppress warnings globally you need to right-click on the member you want to suppress warnings for and select **Suppress *diagnostic ID***, then click **in Suppression File**.
+
+!["Screenshot of WebClient API with green squiggly line and light bulb on the left"](media/api-analyzer/suppress-in-sup-file.jpg)
+
+A file **GlobalSuppressions.cs** will be created in your project. 
```

Comment:
A file **GlobalSuppressions.cs** will be created in your project.  ->
A *GlobalSuppressions.cs* file is added to your project after the first suppresion. (we use italics for file names)